django_sage_meta/models/story.py

```diff
@@ -38,8 +38,8 @@ class Story(AdditionalDataMixin):
     )
     account = models.ForeignKey(
         "InstagramAccount",
-        verbose_name=_("Instagram Story User"),
-        related_name="stories",
+        verbose_name=_("Account used for publishing the story"),
```

Comment:
too long Verbose name. make help_text more meaningful and long then verbose name must be short.